coss/opensource_clubs/models.py

```diff
@@ -129,3 +129,16 @@ class FAQPage(Page):
     content_panels = Page.content_panels + [
         StreamFieldPanel('faq'),
     ]
+
+
+class ActivitiesPage(Page):
+    """Activities Page for Open Source Clubs."""
+    description = RichTextField(blank=True)
+    activity = StreamField([
+        ('info', CardContentBlock(required=True, template='blocks/activity_card.html'),)
```

Comment:
Why don't you add the `template` in the `Meta` class of the stremfield?

Reply:
We could do that, but I thought that this card block can also be used for other cases where we may have slightly different UI. So it's probably better to maintain that flexibility.

Reply:
My guess is that even if you have a template `foo` in the meta class, adding a template `bar` here would override it. Is this true? 
Moreover, having a default template in the Meta class is easier and less prone to errors than having to remember every time to declare one. If you have specific needs that the default doesn't cover then you will definitely remember to override it.